src/System.Net.Sockets.Legacy/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -51,6 +51,7 @@ public SendPacketsAsync(ITestOutputHelper output)
         #region Basic Arguments
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
If none of the tests apply for Nix, shouldn't we only include it for Windows in the csproj?